### PR TITLE
Added Portuguese (pt_PT) translation

### PR DESCRIPTION
**Pull request by rmbb**
New language added - Portuguese (Portugal)

### CODE REVIEW

File: translations/monero-core_pt-pt.ts [hunk 1]
```diff
@@ -0,0 +1,2536 @@
...
+    </message>
+    <message>
+        <location filename="../pages/Settings.qml" line="523"/>
+        <source>Wallet creation height: </source>
+        <translation>Número do bloco de criação da carteira</translation>
+    </message>
+    <message>
+        <location filename="../pages/Settings.qml" line="524"/>
+        <source> &lt;a href=&apos;#&apos;&gt;(Click to change)&lt;/a&gt;</source>
+        <translation"> &lt;a href=&apos;#&apos;&gt;(Clique para alterar)&lt;/a&gt;</translation>
```
Comment (erciccione) [inline translations/monero-core_pt-pt.ts]: `"` got instide the `<translation>` tag and it's breaking the build

File: translations/monero-core_pt-pt.ts [hunk 2]
```diff
@@ -0,0 +1,2536 @@
...
+    <message>
+        <location filename="../pages/Transfer.qml" line="287"/>
+        <location filename="../pages/Transfer.qml" line="290"/>
+        <source>Internal error</source>
+        <translation>Erro interno</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="293"/>
+        <source>No address found</source>
+        <translation>Nenhum endereço en
```
Comment (erciccione) [inline translations/monero-core_pt-pt.ts]: same as above. please remove `"`

File: translations/monero-core_pt-pt.ts [hunk 3]
```diff
@@ -0,0 +1,2536 @@
...
+    </message>
+    <message>
+        <location filename="../pages/Settings.qml" line="498"/>
+        <source>Debug info</source>
+        <translation>Informação de debug</translation>
+    </message>
+    <message>
+        <location filename="../pages/Settings.qml" line="523"/>
+        <source>Wallet creation height: </source>
+        <translation>Número do bloco de criação da carteira</translation>
```
Comment (erciccione) [inline translations/monero-core_pt-pt.ts]: Please add `:` at the end as in the source text

File: translations/monero-core_pt-pt.ts [hunk 4]
```diff
@@ -0,0 +1,2536 @@
...
+- Chaves de transferências
+- Descrições de transferência
+
+O ficheiro cache antigo da carteira será renomeado e poderá ser reposto depois.
+</translation>
+    </message>
+    <message>
+        <location filename="../pages/Settings.qml" line="592"/>
+        <source>Wallet log path: </source>
+        <translation>Path para o ficheiro log da carteira</translation>
```
Comment (erciccione) [inline translations/monero-core_pt-pt.ts]: Please add :  at the end of the string

File: translations/monero-core_pt-pt.ts (hunk 2 above)
Comment (erciccione) [inline translations/monero-core_pt-pt.ts]: you forgot the %2 place marker here

**Comment by erciccione** [comment]
Old PR: #1138

**Comment by rmbb** [comment]
Requested changes have been made. Thanks.

**Comment by EvertonMelo** [comment]
@rmbb 
@erciccione 

this code not running on windows.

https://github.com/monero-project/monero-gui/pull/1141

**Comment by erciccione** [comment]
Hey @rmbb Can you rebase and than squash your commits please?

**Comment by erciccione** [comment]
You have some conflicting files, so you need to [rebase](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request). Squash means unify those commits from 6 to just one. I can help you doing that on our chat `#monero-translations`, but we need to go through git's workflow. If you want, you can add me as collaborator to your repo and i can fix the conflicts and squash for you

**Comment by rmbb** [comment]
Thanks man! 😉

**Comment by luigi1111** [comment]
@erciccione can we fix those extra tabs?

**Comment by erciccione** [comment]
@luigi1111 I think that's just github freaking out, they look correct locally

**Comment by luigi1111** [comment]
@erciccione it looks the same on my local as it does here.

File: lang/languages.xml [hunk 5]
```diff
@@ -32,6 +32,8 @@ List of available languages for your wallet's seed:
         <language display_name="日本語" locale="ja_JP" wallet_language="日本語" flag="/lang/flags/japan.png" qs="none"/>
         <language display_name="Nederlands" locale="nl_NL" wallet_language="Nederlands" flag="/lang/flags/netherlands.png" qs="none"/>
         <language display_name="Polski" locale="pl_PL" wallet_language="English" flag="/lang/flags/poland.png" qs="none"/>
+        <!-- <language display_name="Polski" locale="pl_PL" wallet_language="English" flag="/lang/flags/poland.png" qs="none"/> -->
+	       <language display_name="Português (PT)" locale="pt-pt_PT" wallet_language="Português" flag="/lang/flags/portugal.png" qs="none"/>
```
Comment (luigi1111) [inline lang/languages.xml]: Tab here.

File: monero-wallet-gui.pro [hunk 6]
```diff
@@ -350,6 +350,7 @@ TRANSLATIONS =  \ # English is default language, no explicit translation file
                 $$PWD/translations/monero-core_rs.ts \ # Serbian
                 $$PWD/translations/monero-core_cat.ts \ # Catalan
                 $$PWD/translations/monero-core_tr.ts \ # Turkish
+			             $$PWD/translations/monero-core_pt-pt.ts \ # Portuguese (Portugal)
```
Comment (luigi1111) [inline monero-wallet-gui.pro]: 3 tabs here.

File: qml.qrc [hunk 7]
```diff
@@ -118,6 +118,7 @@
         <file>lang/flags/netherlands.png</file>
         <file>lang/flags/palestine.png</file>
         <file>lang/flags/poland.png</file>
+	 <file>lang/flags/portugal.png</file>
```
Comment (luigi1111) [inline qml.qrc]: Tab here that should be spaces.

**Pull request merged**